dll/win32/shell32/shlexec.cpp

```diff
@@ -2383,14 +2421,63 @@ ShellExecuteExA(LPSHELLEXECUTEINFOA sei)
 
 /*************************************************************************
  * ShellExecuteExW                [SHELL32.293]
- *
  */
 BOOL
 WINAPI
 DECLSPEC_HOTPATCH
 ShellExecuteExW(LPSHELLEXECUTEINFOW sei)
 {
-    return SHELL_execute(sei, SHELL_ExecuteW);
+    HRESULT hrCoInit;
+    DWORD dwError;
+    ULONG fOldMask;
+
+    hrCoInit = SHCoInitializeAnyApartment();
+
+    if (sei->cbSize != sizeof(SHELLEXECUTEINFOW))
+    {
+        dwError = ERROR_ACCESS_DENIED;
+        sei->hInstApp = (HINSTANCE)ERROR_ACCESS_DENIED;
+    }
+    else
+    {
+        fOldMask = sei->fMask;
+
+        sei->fMask |= 0x800;
```

Comment:
Which SEE_ is this and what is it supposed to do?

Reply:
This bit is undocumented.

Reply:
Deleted.

Reply:
If you know Windows uses it then we should probably use it as well but we should at least try to understand what it does so we can invent a name for it.

Reply:
There is no reference for that flag. I think it is useless.